From e37ccb023cc3124e1818b378465860acfb113f13 Mon Sep 17 00:00:00 2001
From: Matthias Kaehlcke <mka@chromium.org>
Date: Wed, 21 Mar 2018 13:47:20 -0700
Subject: [PATCH] perf: Don't install self-tests

There is no point in wasting space in the rootfs with the self-tests.

Signed-off-by: Matthias Kaehlcke <mka@chromium.org>
---
 tools/perf/Makefile.perf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/perf/Makefile.perf b/tools/perf/Makefile.perf
index 91ef44bfaf3e..a560ae569dc4 100644
--- a/tools/perf/Makefile.perf
+++ b/tools/perf/Makefile.perf
@@ -781,7 +781,7 @@ install-tests: all install-gtk
 		$(INSTALL) -d -m 755 '$(DESTDIR_SQ)$(perfexec_instdir_SQ)/tests/shell/lib'; \
 		$(INSTALL) tests/shell/lib/*.sh '$(DESTDIR_SQ)$(perfexec_instdir_SQ)/tests/shell/lib'
 
-install-bin: install-tools install-tests install-traceevent-plugins
+install-bin: install-tools install-traceevent-plugins
 
 install: install-bin try-install-man
 
-- 
2.17.0.rc0.231.g781580f067-goog

